helpers/shard.go

```diff
@@ -0,0 +1,33 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/filecoin-project/dagstore"
+	"github.com/filecoin-project/dagstore/shard"
+)
+
+// AcquireShardSync blocks until the requested shard is acquired or the given context expires.
+// It returns the acquired shard if the acquire was successful. It is the caller's responsibility
+// to close the returned accessor when done.
+func AcquireShardSync(ctx context.Context, dagst *dagstore.DAGStore, sk shard.Key) (*dagstore.ShardAccessor, error) {
+	ch := make(chan dagstore.ShardResult, 1)
+
+	if err := dagst.AcquireShard(ctx, sk, ch, dagstore.AcquireOpts{}); err != nil {
```

Comment:
To make this a true utility function with no assumptions built-in, you should pass `AcquireOpts` as an argument.

Reply:
Done.